src/index.html

```diff
@@ -35,19 +35,19 @@
     <script type="application/javascript" src="xorigin.js"></script>
 
     <link rel="stylesheet" type="text/css" href="thirdparty/CodeMirror/lib/codemirror.css">
-    <!--(if target dev)><!-->
+    <!--(if target dev)-->
```

Comment:
I wonder if these should be like they were in the original? The changes make sense though. @ficristo ?

Reply:
Those comments are used by https://github.com/changer/grunt-targethtml
The changes here should be reverted.

Reply:
Yep, that's what I though, thanks @ficristo !